generators/client/templates/react/src/main/webapp/app/config/notification-middleware.ts.ejs

```diff
@@ -23,7 +23,7 @@ import { toast } from 'react-toastify';
 const addErrorAlert = (message, key?, data?) => {
   <%_ if (enableTranslation) { _%>
   key = key ? key : message;
-  toast.error(translate(key, data));
+  toast.error(translate(key, data), { className: key });
```

Comment:
what are the class names for?

Reply:
if its for e2e cant we select the toast container and get the last child from it? thats how I do in my app, let me know if you need help

Reply:
yes I wanted to give toasts a unique 'id' (the error message key) when using i18n in order to retrieve them easily but indeed, doing this not involves the best changes.
The problem was that i wanted to assert that the right toast appears and not only any toast, especially for i18n, what do you think ?